### DISCUSSION TITLE
Verify strawberry on flathub

### DISCUSSION THREAD

**Discussion by Lunarequest** [strawberrymusicplayer/strawberry#1201] [General]
hi so I'd like to get the flatpak verified for flathub, this would involve adding a file to `.well-known` with a specific token and a specific string to  strawberrymusicplayer.org

**Comment by jonaski** [comment]
Just send me the token and I'll add it

**Reply by Lunarequest** [reply]
this url https://strawberrymusicplayer.org/.well-known/org.flathub.VerifiedApps.txt should return 75d97d5f-7467-428b-a00e-36c8aba159f1

**Reply by jonaski** [reply]
It's done

**Reply by Lunarequest** [reply]
Strawberry music player is mow verified!
